skfda/inference/anova/_anova_oneway.py

```diff
@@ -203,26 +205,35 @@ def _anova_bootstrap(
             )
 
     # List with sizes of each group
-    sizes = [fd.n_samples for fd in fd_grouped]
+    sizes = [fdg.n_samples for fdg in fd_grouped]
 
     # Instance a random state object in case random_state is an int
     random_state = validate_random_state(random_state)
 
-    if equal_var:
```

Comment:
Now, the `cov` method does not implicitly return a matrix, it needs to have the points of evaluation specified. Therefore, specifying a grid beforehand is required. However, in case of dealing with `FDataBasis`, there is no default grid. Thus, I added a default number of points of evaluation (features) in order to retain the same procedure to extract `grid_points`.